src/components/FacetSelect.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2018, The Linux Foundation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import Select from 'react-select'
+
+import 'react-select/dist/react-select.css';
+
+const options = [
+  { value: 'core', label: 'Core'},
+  { value: 'data', label: 'Data'},
+  { value: 'dev', label: 'Dev'},
+  { value: 'docs', label: 'Docs'},
+  { value: 'examples', label: 'Examples'},
+  { value: 'tests', label: 'Tests'}
+]
+
+export default class FacetSelect extends Select {
+  
+  static propTypes = {
+    changeHandler: PropTypes.func,
+    defaultFacets: PropTypes.array
+  }
+
+  static defaultProps = {
+  }
+
+  constructor (props) {
+    super(props);
+    this.state = {value: props.defaultFacets}
+    this.facetChange = this.facetChange.bind(this)
+  }
+
+  facetChange (value) {
+    console.log('You\'ve selected:', value)
+    this.setState({value})
+    console.log('Current state:', JSON.stringify(this.state))
+    const { changeHandler } = this.props;
+    changeHandler && changeHandler(value)
+  }
+
+  render () {
+    const { value } = this.state;
+    return (
+      <div className="section">
```

Comment:
curious about this class. What effect you are looking for? I don't see a `section` class anywhere. What happens if you just delete the div altogether?

Reply:
I added this from an example when I was debugging visual issues with the component, didn't even check the styles.